README.md

```diff
@@ -103,9 +103,9 @@ This repo contains subgraphs defined using a set of standardized schemas. These
 | [Compound](https://compound.finance/) | ✅ | 2.0.1 / 1.7.3 / 1.0.0 | [![Compund Ethereum](./docs/images/chains/ethereum.png)](https://thegraph.com/hosted-service/subgraph/messari/compound-v2-ethereum) |
 | [CREAM Finance](https://cream.finance/) | 🛠 | 2.0.1 / 1.1.3 / 1.0.0 | [![CREAM Finance Arbitrum](./docs/images/chains/arbitrum.png)](https://thegraph.com/hosted-service/subgraph/messari/cream-finance-arbitrum) [![CREAM Finance Polygon](./docs/images/chains/matic.png)](https://thegraph.com/hosted-service/subgraph/messari/cream-finance-polygon) [![CREAM Finance BSC](./docs/images/chains/bsc.png)](https://thegraph.com/hosted-service/subgraph/messari/cream-finance-bsc) [![CREAM Finance Ethereum](./docs/images/chains/ethereum.png)](https://thegraph.com/hosted-service/subgraph/messari/cream-finance-ethereum) |
 | [dForce](https://dforce.network) | 🛠 | 2.0.1 / 1.1.3 / 1.0.0 | [![dforce Arbitrum](./docs/images/chains/arbitrum.png)]() [![dforce Avalanche](./docs/images/chains/avalanche.png)]() [![dforce BSC](./docs/images/chains/bsc.png)]() [![dforce Ethereum](./docs/images/chains/ethereum.png)]() [![dforce Optimism](./docs/images/chains/optimism.png)]() [![dforce Polygon](./docs/images/chains/matic.png)]() |
-| [Euler Finance](https://www.euler.finance/) | ✅ | 1.2.1 / 1.0.2 / 1.0.0 | [![Euler Finance](./docs/images/chains/ethereum.png)](https://thegraph.com/hosted-service/subgraph/messari/euler-finance-ethereum) |
-| [Geist Finance](https://geist.finance/markets) | ✅ | 2.0.1 / 1.0.9 / 1.0.0 | [![Geist Protocol](./docs/images/chains/fantom.png)](https://thegraph.com/hosted-service/subgraph/messari/geist-fantom) |
-| [Iron Bank](https://app.ib.xyz/) | ✅ | 2.0.1 / 1.1.1 / 1.0.0 | [![Iron Bank Fantom](./docs/images/chains/fantom.png)](https://thegraph.com/hosted-service/subgraph/messari/iron-bank-fantom) [![Iron Bank Avalanche](./docs/images/chains/avalanche.png)](https://thegraph.com/hosted-service/subgraph/messari/iron-bank-avalanche) [![Iron Bank Ethereum](./docs/images/chains/ethereum.png)](https://thegraph.com/hosted-service/subgraph/messari/iron-bank-ethereum) |
+| [Euler Finance](https://www.euler.finance/) | ✅ | 1.3.0 / 1.1.0 / 1.0.0 | [![Euler Finance](./docs/images/chains/ethereum.png)](https://thegraph.com/hosted-service/subgraph/messari/euler-finance-ethereum) |
+| [Geist Finance](https://geist.finance/markets) | 🛠 | 2.0.1 / 1.0.9 / 1.0.0 | [![Geist Protocol](./docs/images/chains/fantom.png)](https://thegraph.com/hosted-service/subgraph/messari/geist-fantom) |
```

Comment:
should keep the status for geist and iron bank

Reply:
accident, good catch